code/datums/controllers/action_controls.dm

```diff
@@ -149,8 +153,15 @@ var/datum/action_controller/actions
 			bar.pixel_y = 5
 			bar.pixel_x = 0
 			border.pixel_y = 5
-			A.vis_contents += bar
-			A.vis_contents += border
+			if (bar_on_owner)
+				A.vis_contents += bar
+				A.vis_contents += border
+			if (place_to_put_bar)
+				bar.on_target = 1
+				border.on_target = 1
```

Comment:
Defined but never used? Prune?
If they do have a use, better to use `TRUE` than `1`
![image](https://user-images.githubusercontent.com/9563541/124369239-885f2b00-dc1e-11eb-82cd-a6db8bfc0e71.png)

Reply:
Oh yeah that was an accidental left over, my bad